src/controllers/hotkeys/HotkeyController.cpp

```diff
@@ -516,7 +516,7 @@ void HotkeyController::addDefaults(std::set<QString> &addedHotkeys)
                             std::vector<QString>(), "reopen split");
 
         this->tryAddDefault(addedHotkeys, HotkeyCategory::Window,
-                            QKeySequence("Ctrl+H"), "toggleLocalR9K",
+                            QKeySequence("Ctrl+."), "toggleLocalR9K",
                             std::vector<QString>(), "toggle local r9k");
```

Comment:
This does not change configs for existing users. This is probably not what should be done here as CMD+H is already reserved there is no harm forcing `toggle local r9k` to be rebound to it for every user.